unlock-app/src/contexts/AlertContext.ts

```diff
@@ -0,0 +1,18 @@
+import { useContext, createContext } from 'react'
+
+interface AlertContextInterface {
+  openAlert: ({ title: string, body: string }) => void
+}
+
+export const AlertContext = createContext({} as AlertContextInterface)
+
+/**
+ * Hook
+ * @returns
+ */
+export const useAlert = () => {
+  const { openAlert } = useContext(AlertContext)
+  return openAlert
+}
+
+export default AlertContext
```

Comment:
Do we need to store alert state globally? 

Any reason? I think it would be fine to just store it locally.

Reply:
Moved!